### ISSUE TITLE
mrsk fails to connect to servers when net-ssh is < 7.x

### ISSUE THREAD

**Issue by matharvard** [mrsked/mrsk#180]
mrsk wasn't able to connect to target deploy servers using sshkit when the net-ssh gem was at version 6.1.0. net-ssh:7.1.0 worked perfectly fine though. I am wondering if this should be added as a requirement to the mrsk gem itself (I'm not sure exactly what version fixed it, would have to investigate).

For context I had an application with a requirement of net-sftp:3.0.0 which meant that net-ssh was locked to < 7.0.0 due to that version's dependency requirements. mrsk installed just fine as it only specifies an sshkit gem version requirement. The result was that every time mrsk attempted to open a connection to a server, authentication would fail (`ERROR (Net::SSH::AuthenticationFailed): Authentication failed for user root@1.1.1.1`).

**Comment by dhh** [comment]
Yes, please do create a PR listing net-ssh as a dependency with the minimum version that'll work 👍

**Comment by jasonfb** [comment]
1. confirm net-ssh gem is at least 7.1 or higher
2. run this on your terminal:
```
eval `ssh-agent -s`
ssh-add path/to/keyfile
```
